source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -102,6 +102,10 @@ the feature cannot determine what to do and the operation will fail.
 
 * The cluster's management network stays on the original interface which is excluded from the operation. 
 
+* QoS, DNS and custom_properties are currently unsupported.
+
+* Multi-host copying with labels.
```

Comment:
So this is listed as a limitation, yet I failed to realize what's the meaning of this sentence.
Could you maybe elaborate on where the limitation stands regarding labels?

Reply:
@MosheSheena I think it means copying the networks's label. So I guess the network will be copied but the label will not.
Do you think something like "Multi-host network labels copying" will be better?

Reply:
@almusil could you please help us out on the label's part? to my knowledge labels are not supported, and this was not covered by QE.

Bell I believe these limitations should explicitly and clearly state what is not supported and in what way meaning if copying labels isn't supported then it should be written as plain as possible so that users could quickly be informed of it.

Reply:
AFAIR labels were never intended to be copied because they are kinda separate unit. So it is fine to mention that copying of labels is not supported but I don't quite understand the multi-host part.

Reply:
@bellle I believe it should be strictly written that the following will not be copied from the source host to the target host: QoS, DNS, custom_properties (without hints to future support).
Labels attached to network interfaces on the source host will not be attached to network interfaces on the target host.

Reply:
Ok I made it verbose to be able to understand.